### DISCUSSION TITLE
Can't build 64 bit apk for AIR 33

### DISCUSSION THREAD

**Discussion by mistergreen** [airsdk/Adobe-Runtime-Support#643] [Q&A]
I'm using Flash Builder, OS X 10.14.6. I tried using the added -arch armv8 launch parameter to build but it won't build so I modified adt.cfg (DefaultArch=armv8 & OverrideArch=armv8)

All I see in the package is a armeabi-v7a folder.

I'm using AIR 33.0.2.388

**Comment by mistergreen** [comment]
Got it to build 64-bit through ADT command line. Looks like Flash Builder is the culprit.

**Comment by mistergreen** [comment]
Ah, found the problem... I went to see the command FB is building. **Notice -d32**. FB is forcing a 32-bit build. I have no idea how to change that. I guess I'll just copy the command line and build manually.

> /usr/bin/java -d32 -jar /Applications/Adobe Flash Builder 4.7/eclipse/plugins/com.adobe.flash.compiler_4.7.0.349722/AIRSDK/lib/adt.jar -package -target apk-captive-runtime -arch armv8 -storetype pkcs12 -keystore /Users/cccc/Documents/Test33/certificat_android.p12 -storepass <Value cannot be displayed> Test33.apk Test33-app.xml Test33.swf

**Reply by ugheddu** [reply]
Hi, how you  "...see  the command FB is building..." ? 

Thanks,

**Reply by mistergreen** [reply]
I haven't used FB in a long time, check the 'build' option.
